hyperspy/roi.py

```diff
@@ -980,6 +1069,12 @@ def __init__(self, x1, y1, x2, y2, linewidth=0):
         self.x1, self.y1, self.x2, self.y2 = x1, y1, x2, y2
         self.linewidth = linewidth
 
+    def __getitem__(self, *args, **kwargs):
+        _tuple = (self.x1, self.y1, self.x2, self.y2, self.linewidth)
+        _tuple = (self.x1, self.y1, self.x2, self.y2)
```

Comment:
I guess this line should be removed?

Reply:
Yes, thanks. It turns out that I forgot to add tests for Circle and Line2D. After adding them in the last commit I found 1 more bug of course...